test/gtest/uct/test_uct_perf.cc

```diff
@@ -42,6 +42,24 @@ const test_perf::test_spec test_uct_perf::tests[] =
     ucs_offsetof(ucx_perf_result_t, msgrate.total_average), 1e-6, 0.8, 80.0,
```

Comment:
minor: add `short` to the name for `UCT_PERF_DATA_LAYOUT_SHORT` tests, e.g. `am short latency`